applications/xe-cnf/.gitignore

```diff
@@ -0,0 +1,53 @@
+# These are some examples of commonly ignored file patterns.
```

Comment:
This file doesn't have Copyright header. This is git configuration file. There is no need to have this file fin repo. Can you please remove this file?

Reply:
Please remove this file